### ISSUE TITLE
Question about the loss

### ISSUE THREAD

**Issue by shinpaul14** [CAMMA-public/rendezvous#25]
How did you generate the weight for the tool, verb, and target loss function?

**Comment by nwoyecid** [comment]
Training dataset statistics. Please refer to the paper.
